azure/template.json

```diff
@@ -0,0 +1,199 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "resourceEnvironmentName": {
+            "type": "string",
+            "metadata": {
+                "description": "Short name of the environment. Used for the name of resources created"
+            }
+        },
+        "deployStubFunctions": {
+            "type": "bool",
+            "defaultValue": false
+        },
+        "aseResourceGroup": {
+            "type": "string",
+            "defaultValue": ""
+        },
+        "aseHostingEnvironmentName": {
+            "type": "string",
+            "defaultValue": ""
+        },
+        "appServicePlanSize": {
+            "type": "string",
+            "allowedValues": [
+                "1",
+                "2",
+                "3"
+            ],
+            "defaultValue": "1"
+        },
+        "appServicePlanInstances": {
+            "type": "int",
+            "defaultValue": 2
+        }
+    },
+    "variables": {
+		"deploymentUrlBase": "https://raw.githubusercontent.com/SkillsFundingAgency/das-platform-building-blocks/master/templates/",
+        "resourcePrefix": "[concat('das-',toLower(parameters('resourceEnvironmentName')),'-rsrvwkr')]",
+        "appServicePlanName": "[concat(variables('resourcePrefix'),'-asp')]",
+        "functionAppName": "[concat(variables('resourcePrefix'),'-refreshcourse-fa')]",
+        "storageAccountName": "[concat('das',toLower(parameters('resourceEnvironmentName')),'rsrvwkrstr')]"
+    },
+    "resources": [
+        {
+            "apiVersion": "2017-05-10",
+            "name": "WorkerAppServicePlan",
```

Comment:
deployment names should be lower-case-with-hyphens -> https://github.com/SkillsFundingAgency/das-platform-building-blocks#property-name-formatting